pkg/hooks/audit_log_file.go

```diff
@@ -0,0 +1,45 @@
+package hooks
+
+import "github.com/square/p2/pkg/logging"
+
+// Returns a FileAuditLogger using the given logger
+func NewFileAuditLogger(logger *logging.Logger) *FileAuditLogger {
```

Comment:
the word `File` is a bit confusing to me here since it could just be logging to stdout, maybe i'm not unix-y enough (everything is a file)